crates/bevy_render/src/color/mod.rs

```diff
@@ -146,8 +146,8 @@ impl Color {
     ///
     /// See also [`Color::rgba`], [`Color::rgb_u8`], [`Color::hex`].
```

Comment:
Some places you have renamed the type in the docs, other places you haven't. I think it's best to be consistent.

I'm not sure what is easiest from a porting perspective, but for reviewing it's easier to check that they haven't changed instead of searching for ones you've missed.